src/Content/Version.php

```diff
@@ -120,7 +120,7 @@ public function create(
 		VersionRules::create($this, $fields, $language);
 
 		// track the changes
-		if ($this->id->is(VersionId::changes()) === true) {
+		if ($this->id->is(VersionId::CHANGES) === true) {
```

Comment:
Why do we use the constants here, but not for the sibling calls?

Reply:
I used the constants here as this was used in most other `$versionId::is()` calls - and as comparing it there to the constant felt the proper thing to do in context of the VersionId class, where no `VersionId::from()` call ensures proper translation.

`Version::sibling()` and `ModelWithContent::version()` receive strings as they ensure a proper handling by calling `VersionId:;from()` internally.

Reply:
Reworked `VersionId::is()` a little bit as suggested.